script.js

```diff
@@ -1,4 +1,586 @@
+
+  if (userInput == `standard`) {
+    message = `You are now in the standard game mode. <br> <br> Please type 'scissors', 'paper' or 'stone' to start the standard SPS game with me!`;
+  }
+  if (gameResult == `draw`) {
+    message = `It's a draw! <br> <br> This is the standard game. <br> <br> We both chose ${programSelection}! <br> You didn't copy my answer did you? 😜`;
+  }
+  if (gameResult == `win`) {
+    message = `You won! <br> <br> This is the standard game. <br> <br> You chose ${userInput} and I chose ${programSelection}. <br> Good job!`;
+  }
+  if (gameResult == `lose`) {
+    message = `You lost! <br> <br> This is the standard game. <br> <br> You chose ${userInput} and I chose ${programSelection}. <br> Better luck next time!`;
+  }
+  return message;
+};
+
+// Create a function to update winning player for the last round
+var updateWinner = function (userInput, programSelection) {
+  var winner = `not yet assigned`;
+  gameResult = result(userInput, programSelection);
+  if (gameResult == `win`) {
+    winner = `YOU are`;
+  }
+  if (gameResult == `lose`) {
+    winner = `I am`;
+  }
+  console.log(`${winner} the winner from last round.`);
+  return winner;
+};
+
+// Create a function for reversed game mode
+var koreanGame = function (
+  userInput,
+  programSelection,
+  gameResult,
+  lastWonPlayer
+) {
+  var message = `Sorry, I didn't get that. <br> <br> Please type 'scissors', 'paper' or 'stone' to start playing the game with me 😊 <br> <br> You can also type 'korean' to try our korean mode and 'standard' for the original game. <br> <br> --- What is the korean version of SPS? --- <br> In Korea, a two-player upgraded version exists by the name muk-jji-ppa. After showing their hands, the player with the winning throw shouts "muk-jji-ppa!" upon which both players throw again. If they throw differently (for example, rock and paper, or paper and scissors), whoever wins this second round shouts "muk-jji-ppa!" and thus the play continues until both players throw the same item (for example, rock and rock), at which point whoever was the last winner becomes the actual winner.`;
+
+  if (userInput == `korean`) {
+    message = `You are now in the korean game mode. <br> <br> Please type 'scissors', 'paper' or 'stone' to start the korean SPS game with me! <br> <br> --- How to win? --- <br> Once both players threw the same item, whoever won the last round becomes the actual winner.`;
```

Comment:
Good thinking for the user, in breaking up the messages into two parts,
one in the original message, and the second one here